README.md

```diff
@@ -0,0 +1,26 @@
+# Mars Rover
+
+## Assignment Requirements
+
+For this "Mars Rover" assignment, you must accomplish the following:
+
+1. Use unit testing to validate the behavior of the constructor within an
+   existing class.
+1. Use test driven development (TDD) to guide the creation of new classes and
+   methods.
+
+## How Your Assignment is Graded
+
+First, your TA is going to check your code.
+They are looking to see if you have properly coded the `Message` class so that the tests pass.
+They will then turn their attention to the `Rover` class. Here, you need to complete the tests for the `Rover` class and add some functionality to the class itself. 
+
+After they review your code, they will run the code as-is to verify that you have completed all the tests and that all the tests pass.
+You should have 13 tests that pass. If you have completed the bonus mission, you should have 14 tests that pass.
+
+### Grading Tests
+
+After these initial checks, your TA will run the tests inside `studentgrade.spec.js`. You may have noticed this file already, but this file only contains grading tests for your TA to use.
```

Comment:
If this file is already in the spec folder, aren't these tests already being run?  Do we need to remove this file from the folder to begin with? 

Instructions for TA's from gitlab -> https://gitlab.com/LaunchCodeEducation/github-classroom-autograded-assignments/js-unit-1/mars-rover

Reply:
I added more clarification for students on how to insert it. The TAs would know this as they are industry professionals.